### ISSUE TITLE
Label does not apply to to inner input on kendo file select

### ISSUE THREAD

**Issue by AndrewJohnWalker** [telerik/kendo-angular#4445]
### Description

https://github.com/telerik/kendo-angular/issues/3022 - Seems to possibly be a duplicate/regression of this? 

The WAVE accessibility tool raises that a form input is missing a label, and it seems impossible to label the inner input.

### Steps To Reproduce

https://codesandbox.io/p/sandbox/twilight-bird-xxcp3n?workspaceId=8f7f9ef6-74cc-4d4d-b9c0-3ecec2911aca

Wrap a kendo fileselect in a kendo-label

### Screenshots or video

![image](https://github.com/user-attachments/assets/07058067-0d42-4737-ba13-51a671046afc)

### Actual Behavior

Inner input cannot be labeled.

### Expected Behavior

Inner input is labelled.

### Browser

All

### Browser version

latest

**Comment by pkeridactyl** [comment]
I'm having the same issue. Has there been any movement on this?
